api/include/opentelemetry/logs/logger.h

```diff
@@ -39,75 +39,95 @@ class Logger
   virtual ~Logger() = default;
 
   /* Returns the name of the logger */
-  virtual nostd::string_view getName() = 0;
+  // TODO: decide whether this is useful and/or should be kept, as this is not a method required in the specification.
+  // virtual nostd::string_view getName() = 0;
 
   /**
-   * Creates a log message with the passed characteristics.
+   * Each of the following overloaded log(...) methods 
+   * creates a log message with the specific parameters passed. 
    *
    * @param name the name of the log event.
-   * @param sev the severity level of the log event.
-   * @param msg the string message of the log (perhaps support std::fmt or fmt-lib format).
+   * @param severity the severity level of the log event.
+   * @param message the string message of the log (perhaps support std::fmt or fmt-lib format).
    * @param record the log record (object type LogRecord) that is logged.
    * @param attributes the attributes, stored as a 2D list of key/value pairs, that are associated
    * with this log.
    * @throws No exceptions under any circumstances.
    */
 
-  // Log issue about macros (disabling log events below certain level)
-
   /* The below method is a logging statement that takes in a LogRecord.
    * A default LogRecord that will be assigned if no parameters are passed to Logger's .log() method
    * which should at minimum assign the trace_id, span_id, and timestamp
    */
   virtual void log(const LogRecord &record) noexcept = 0;
 
-  // First, finalize core,
-  // Afterwards, come back and add overloads
 
   /** Overloaded methods for unstructured logging **/
-  void log(nostd::string_view msg) noexcept
+  void log(nostd::string_view message) noexcept
```

Comment:
inline? @maxgolov @ThomsonTan what do you think?

Reply:
Yes.